### PR TITLE
Release multiple tags

### PR DESCRIPTION
**Pull request by cpuguy83**
- Modify the fronted job to support a comma separated tag list
- Modify release workflow to send multiple tags to the build job:
  - `latest`
  - `<major>.<minor>`
  - `<major>.<minor>.<patch>`

### CODE REVIEW

**Comment by cpuguy83** [comment]
I'm actually getting a 403 error from ghcr when it tries to push.
Not sure if that's some flakiness with github.
Ref: https://github.com/cpuguy83/dalec-test-fork/actions/runs/8058134256/job/22010622326#step:6:244

**Comment by cpuguy83** [comment]
Ok, seems like it was a github issue.
I think possibly because I deleted that repo earlier today just to recreate it to test this.

Ran through, succssfully, a couple of times in a new repo name:

https://github.com/cpuguy83/dalec-test-release/actions/runs/8059805774
https://github.com/cpuguy83/dalec-test-release/actions/runs/8059833956

image

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -14,19 +14,30 @@ permissions:
...
-      TRIMMED_REF_NAME: ${{ steps.trim.outputs.TRIMMED_REF_NAME }}
+      tags: ${{ steps.tags.outputs.OUTPUT_TAGS }}
     steps:
-      - name: Trim tag ref
-        id: trim
-        run: echo "TRIMMED_REF_NAME=${REF_NAME#v}" >> "$GITHUB_OUTPUT" && cat "$GITHUB_OUTPUT"
+      - name: Get image tag list
+        id: tags
+        run: |
+          set -eux -o pipefail
```
Comment (sozercan) [inline .github/workflows/release.yml:25]: do we want to make this a default (except debug of course) by using `shell: bash` everywhere?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -14,19 +14,30 @@ permissions:
...
+
+          # Remove leading "v" from tag name
+          TRIMMED_REF_NAME="${REF_NAME#v}"
+
+          # Creates a commaa separated list of tags
```
Comment (sozercan) [inline .github/workflows/release.yml:30]:
```suggestion
          # Creates a comma separated list of tags
```

**Review by sozercan — APPROVED** [review]
LGTM

File: .github/workflows/release.yml (hunk 1 above)
Comment (cpuguy83) [reply .github/workflows/release.yml:25]: That's fine, though I'd probably still make it explicit.

File: .github/workflows/release.yml (hunk 2 above)
Comment (cpuguy83) [reply .github/workflows/release.yml]: Fixed

File: .github/workflows/release.yml (hunk 1 above)
Comment (cpuguy83) [reply .github/workflows/release.yml:25]: Added `shell: bash`

**Pull request merged**